azurerm/internal/services/datafactory/data_factory_linked_service_odata_resource_test.go

```diff
@@ -0,0 +1,165 @@
+package datafactory_test
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
```

Comment:
Could we also write a complete test that sets all properties? and then an upgrade test that goes from a basic config -> complete -> basic ?

Reply:
How are the integration tests run ? I don't have information concerning an OData endpoint open with othere type of authentication other than anonyme and basic auth